src/charm.py

```diff
@@ -187,6 +198,10 @@ def _setup_cron_job(self, config):
         with open("/etc/cron.d/{}".format(self.model.app.name), "w") as f:
             f.write("{} root apt-mirror\n".format(config["cron-schedule"]))
 
+    def _remove_cron_job(self):
```

Comment:
**nitpick:** 
It will be wired if we just change the file to empty. Someone look at this folder will find a empty file without any other information.
Maybe we can delete it or print some debug log?

Reply:
Agree, I removed the cron job instead.